ci/apache-arrow.rb

```diff
@@ -0,0 +1,57 @@
+class ApacheArrow < Formula
```

Comment:
Having the `Formula` under the `ci/` directory it a bit odd to me, I'd expect it to be under `dev/tasks/brew-formula` like the rest of the binary packages. Not mentioning it's naming, it doesn't reflect that it is related to brew.

Reply:
I'd move it under 'dev/tasks/bre-formula` and update the release script below (which already updates the files under `dev/tasks/linux-packages`)

Reply:
I see now, there is a `dev/tasks/brew-formulae` already. Please move this file there too.